eng/pipelines/prepare-pipelines.yml

```diff
@@ -0,0 +1,16 @@
+trigger: none
+
+pr:
+  branches:
+    include:
+      - master
+      - feature/*
+
+extends:
+  template: /eng/common/pipelines/templates/steps/prepare-pipelines.yml
```

Comment:
Do we disable the PR triggers by default? That's the step which is easy to overlook but affect other PRs.

Reply:
The pipeline definition for this YAML will be created manually for each repo. In that configuration we disable automatically running without a comment from a team member. One is to stop it always running, the other is because the pipeline has access to secrets and we only want authorized people to be able to run it.